apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -118,17 +118,12 @@ do_send(Socket, Transport, B, M, T, C, MinRTT) ->
         Cutoff when Cutoff > T ->
             do_send_opt(Socket, Transport, B, M, T, C, []);
         _ ->
-            do_send_opt(Socket, Transport, B, M, T, C, [no_rr])
+            do_send_opt(Socket, Transport, B, M, T, C, [{rr, off}])
```

Comment:
you are right, would also be simplified by combining do_send_opts and do_send, as a result of https://github.com/dalmatinerdb/dproto/pull/8